lib/ClangImporter/ImportDecl.cpp

```diff
@@ -4586,7 +4634,8 @@ namespace {
       if (!importer::hasNativeSwiftDecl(decl))
         return false;
       auto wrapperUnit = cast<ClangModuleUnit>(dc->getModuleScopeContext());
-      swiftDecl = resolveSwiftDecl<T>(decl, name, wrapperUnit);
+      swiftDecl = resolveSwiftDecl<T>(decl, name, /*hasCustomName=*/true,
```

Comment:
Maybe this suggests that "hasKnownSwiftName" would be a better name for the parameter?